### ISSUE TITLE
Omit parameter names in REST Clients section of reference docs

### ISSUE THREAD

**Issue by ethan3011** [spring-projects/spring-framework#32335]
**Affects:** 6.1.4 doc

I just saw `HttpMethod method` parameter in `rest-clients.adoc`.

I think it should be `HttpMethod` only, because other examples just use `HttpMethod`.

thanks

```
execute(String, **HttpMethod method**, RequestCallback, ResponseExtractor, Object...)
execute(String, **HttpMethod method**, RequestCallback, ResponseExtractor, Map)
execute(URI, **HttpMethod method**, RequestCallback, ResponseExtractor)
```
<img width="481" alt="image" src="https://github.com/spring-projects/spring-framework/assets/80470138/27dec6ff-a81d-4543-b564-3fa2d8a15412">

**Comment by ethan3011** [comment]
compare with others
<img width="451" alt="image" src="https://github.com/spring-projects/spring-framework/assets/80470138/77dbdb0c-07b8-4ba2-8ef7-d3b29c70a2e6">

**Comment by sbrannen** [comment]
Hi @ethan3011,

Congratulations on submitting your first issue for the Spring Framework! 👍 

I've addressed your concerns in 5830aac1d43b605c5b56d1a7e52118742c84c063.

Cheers

**Comment by ethan3011** [comment]
Thanks a lot :)
